packages/python/plotly/plotly/tests/test_core/test_px/test_px_functions.py

```diff
@@ -149,10 +149,6 @@ def test_sunburst_treemap_with_path():
     fig = px.sunburst(df, path=path, values="values")
     assert fig.data[0].branchvalues == "total"
     assert fig.data[0].values[-1] == np.sum(values)
-    # Continuous colorscale
```

Comment:
should we keep this one also just for completeness? it still passes right?

Reply:
no, interestingly it does not because this one tests the behaviour we don't want any more (that `color` has the same values as `values`, since we used to have the same `sum` aggregation function but now we have two different functions.)

Reply:
got it, makes sense.